.github/workflows/prettier.yml

```diff
@@ -16,6 +16,6 @@ jobs:
       - name: Prettify code
         uses: creyD/prettier_action@v4.3
         with:
-          prettier_options: --write .
+          prettier_options: --write . --config .prettierrc
```

Comment:
Pretty sure that prettier tries to find the config file, so no need to specify it
<img width="832" alt="image" src="https://github.com/spicetify/spicetify-cli/assets/19473034/43a5ec3f-baaa-483e-90ec-af5a1c8fe7dc">
```suggestion
          prettier_options: --write .
```

Reply:
The action runs the prettier CLI, therefore you can assume that it does. But feel free to test by changing a parameter of the config ;)